src/Strategy/Python/Setuptools.hs

```diff
@@ -0,0 +1,73 @@
+module Strategy.Python.Setuptools
+  ( discover',
+  )
+where
+
+import Control.Carrier.Output.IO
+import Control.Effect.Diagnostics (Diagnostics)
+import qualified Control.Effect.Diagnostics as Diag
+import Control.Monad.IO.Class
+import Data.Foldable (find)
+import Data.List (isInfixOf, isSuffixOf)
+import Discovery.Walk
+import Effect.ReadFS
+import Graphing (Graphing)
+import Path
+import qualified Strategy.Python.ReqTxt as ReqTxt
+import qualified Strategy.Python.SetupPy as SetupPy
+import Types
+
+discover' :: MonadIO m => Path Abs Dir -> m [NewProject]
+discover' dir = map mkProject <$> findProjects dir
+
+findProjects :: MonadIO m => Path Abs Dir -> m [SetuptoolsProject]
+findProjects = walk' $ \dir _ files -> do
+  let reqTxtFiles =
+        filter
+          ( \f ->
+              "req" `isInfixOf` fileName f
+                && ".txt" `isSuffixOf` fileName f
+          )
```

Comment:
Ormolu did this, because I inserted a line break (I thought the line was too long). The spacing is odd here, though I'm not sure let-bindings improve readability 😕. I'll see if I can improve the spacing somehow